tests/Config/Parser/NoneTest.php

```
@@ -0,0 +1,59 @@
+        $this->assertSame(3.14159, $data["float"]);
+        $this->assertSame(-50, $data["negative"]);
+        $this->assertTrue($data["boolean_true"]);
+        $this->assertFalse($data["boolean_false"]);
+        $this->assertNull($data["null"]);
+    }
+
+    public function testNoneParseException(): void
+    {
+        $this->expectException(Parse::class);
+
+        $data = $this->parser->parse(null);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$data'. (undefined)

(UnusedLocalVariable)